clang/lib/3C/ProgramInfo.cpp

```diff
@@ -483,6 +488,16 @@ bool ProgramInfo::insertNewFVConstraint(FunctionDecl *FD, FVConstraint *FVCon,
     // external method.
     Ret = insertIntoExternalFunctionMap(ExternalFunctionFVCons, FuncName, FVCon,
                                         FD, C);
+    bool IsDef = FVCon->hasBody();
+    if (IsDef) {
+      // Some function (e.g, atoi) are given definitions inside the system
+      // headers. Here we ensure that such functions are treated the same as
+      // function without definitions.
```

Comment:
Why does it make sense to do so?

Reply:
(the answer thus answers my questions about explicitly defined functions, to this point.)